challengeutils/mirrorwiki.py

```diff
@@ -52,6 +52,9 @@ def mirrorwiki(syn, entity, destination, force_merge=False):
                     entity_project_and_wiki_id = "%s/wiki/%s" % (entity.id, entity_page_id)
                     destination_project_and_wiki_id = "%s/wiki/%s" % (destination.id, wiki_mapping[entity_page_id])
                     destination_wiki.markdown=re.sub(entity_project_and_wiki_id, destination_project_and_wiki_id, destination_wiki.markdown)
+                    entity_project_and_wiki_id = "{}%2Fwiki%2F{}".format(entity.id,entity_page_id)
```

Comment:
why is this done twice (see line 52)?

Reply:
They are different.  When you insert a button widget, it gets translated to `synid%2Fwiki%2Fwikiid`, so I have to add a second replacement.  Granted, it could definitely be named better.